Userland/Libraries/LibTTF/Font.cpp

```diff
@@ -246,6 +246,11 @@ RefPtr<Font> Font::load_from_offset(ByteBuffer&& buffer, u32 offset)
         return nullptr;
     }
 
+    if (buffer.size() < offset + (u32)Offsets::NumTables + sizeof(u16) - 1) {
```

Comment:
`offset` is input-controlled here, right? So the expression `offset + (u32)Offsets::NumTables + sizeof(u16) - 1` could overflow

Reply:
Indeed. In fact, the only way this issue can happen in the first place is because the offset in the file was 0xFFFFFFFB, which caused an overflow on the check above (`offset + (u32)Sizes::OffsetTable`) making it think it had enough data when it didn't. Should I just add an overflow check for `offset + (u32)Sizes::OffsetTable` instead?

Reply:
That sounds sensible :)